packages/compass-components/src/components/options-dropdown-button.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import { Link, Icon } from './leafygreen';
+import { css } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const optionsButtonStyles = css({
+  backgroundColor: 'transparent',
+  border: 'none',
+  display: 'inline',
+  height: spacing[4] + spacing[1],
+  ':focus': {
+    outline: `${spacing[1]}px auto ${uiColors.focus}`,
+  },
+});
+
+const optionStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
```

Comment:
Doesn't look like this wrapper is needed really, can it be just a button that controls the layout of components inside?

Reply:
I think we want it to make the elements inside of the button vertically aligned, passing these directly to the link/button they appear to be overwritten. Totally open to other ways to do that.